fe/fe-core/src/main/java/com/starrocks/planner/OlapScanNode.java

```diff
@@ -977,6 +977,8 @@ protected void toThrift(TPlanNode msg) {
             }
 
             msg.olap_scan_node.setUse_pk_index(usePkIndex);
+            msg.olap_scan_node.setEnable_prune_column_after_index_filter(
```

Comment:
Does lake table also support this? If so, this also need to be add in the `if (olapTable.isCloudNativeTableOrMaterializedView())` branch.